app/services/organization/organization.service.go

```diff
@@ -0,0 +1,92 @@
+package user
+
+import (
+	"context"
+	model "database-ms/app/models"
+	"database-ms/config"
+	"database-ms/utils"
+
+	"github.com/google/uuid"
+	mgo "gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+type OrganizationServiceInterface interface {
+	Create(context.Context, *model.Organization) error
+	// FindByThingId(context.Context, string) ([]*model.Organization, error)
+	Update(context.Context, string, *model.Organization) error
+	Delete(context.Context, string) error
+	GetOrganizations(context.Context, string) ([]*model.Organization, error)
+}
+
+type OrganizationService struct {
+	db     *mgo.Session
+	config *config.Configuration
+}
+
+func NewOrganizationService(db *mgo.Session, c *config.Configuration) OrganizationServiceInterface {
+	return &OrganizationService{config: c, db: db}
+}
+
+func (service *OrganizationService) Create(ctx context.Context, organization *model.Organization) error {
+	organization.ID = bson.NewObjectId()
+	organization.ApiKey = uuid.NewString()
+	return service.addOrganization(ctx, organization)
+}
+
+func (service *OrganizationService) FindUpdatedOrganization(ctx context.Context, thingId string, lastUpdate int64) ([]*model.Organization, error) {
```

Comment:
I don't think it is necessary to have a FindUpdatedOrganizations since they don't have a lastUpdated property

Reply:
That's true, I'll double check what DB operations we need exactly for both organizations and users.